srml/executive/src/lib.rs

```diff
@@ -301,7 +301,10 @@ mod tests {
 	};
 	use support::{
 		impl_outer_event, impl_outer_origin, parameter_types, impl_outer_dispatch,
-		traits::{Currency, LockIdentifier, LockableCurrency, WithdrawReasons, WithdrawReason},
+		traits::{
+			Currency, LockIdentifier, LockableCurrency, WithdrawReasons, WithdrawReason,
+			ExistenceRequirement
```

Comment:
```suggestion
			ExistenceRequirement, 
```

and place in one line up to 100 chars if possible